### PR TITLE
Added PG Example to Databases Documentation (Rocket Contrib)

### PR DESCRIPTION
**Pull request by PrismaPhonic**
This PR adds a small example of connecting Postgres to the databases documentation, so that it's obvious how trivial it is to tie a non-sqlite database to Rocket.

### CODE REVIEW

File: contrib/lib/src/databases.rs [hunk 1]
```diff
@@ -203,6 +210,21 @@
...
 //!
+//! Tieing your application to a different RDBMS is simple, and follows the same pattern.
+//! Here's an example using Postgres with Diesel as the driver:
+//!
+//! ```rust
+//! # extern crate rocket;
+//! # #[macro_use] extern crate rocket_contrib;
+//! # #[cfg(feature = "diesel_postgres_pool")]
+//! # mod test {
+//! use rocket_contrib::databases::diesel;
```
Comment (jebrosen) [inline contrib/lib/src/databases.rs]: Can we use a non-Diesel driver here, just so it's not too similar?

Comment (PrismaPhonic) [reply contrib/lib/src/databases.rs]: Sure, I'll go ahead and make the adjustment

Comment (PrismaPhonic) [reply contrib/lib/src/databases.rs]: @jebrosen just updated with a new commit to use Rust-Postgres as the driver.

**Comment by PrismaPhonic** [comment]
Just realized that prettier auto-formatted the code on save.  Will roll back a commit and turn off auto-save formatting and make adjustments again.  Sorry about that

**Comment by jebrosen** [comment]
Merged in 622f4f3, thank you!

**Pull request closed without merging**